README.md

```diff
@@ -198,6 +198,10 @@ The main breaking change from previous version is that the package has been rena
 
 For other breaking changes, check [the migration guide of vite](https://vitejs.dev/guide/migration.html).
 
+# Testing
+
+If you are using [vitest](https://vitest.dev/), this plugin already injects the necessary configuration for you. It even automatically detects if you have `@testing-library/jest-dom` installed in your project and automatically adds it to the `setupFiles`. All you need to add (if you want) is `globals`, `coverage`, and other testing configuration of your choice. If you can live without those, enjoy using vitest without the need to configure it yourself.
```

Comment:
```suggestion
If you are using [vitest](https://vitest.dev/), this plugin already injects the necessary configuration for you. It even automatically detects if you have `@testing-library/jest-dom` installed in your project and automatically adds it to the `setupFiles`. All you need to add (if you want) is `globals`, `coverage`, `testTransformMode`, and other testing configuration of your choice. If you can live without those, enjoy using vitest without the need to configure it yourself.
```

Reply:
Actually, testTransformMode shouldn't be required in any but edge cases, when the environment does not match the mode for some strange reasons.

Reply:
As I have checked this, it does not cause a problem anymore. Okay, so nevermind that.